### PR TITLE
Move 'FetchKeys()' function into a subpkg

### PR DESCRIPTION
**Pull request by xcoulon**
Signed-off-by: Xavier Coulon <xcoulon@redhat.com>

### CODE REVIEW

**Comment by fabric8cd** [comment]
@xcoulon snapshot fabric8-auth image is available for testing.  `docker pull docker.io/fabric8/fabric8-auth:SNAPSHOT-PR-434-1`

File: token/keycloak/keys/fetch_keys.go [hunk 1]
```diff
@@ -0,0 +1,98 @@
+package keys
```
[the remark this answers is not in the document]
Comment (xcoulon) [reply token/keycloak/keys/fetch_keys.go]: sure! Let me change that.

**Comment by xcoulon** [comment]
@alexeykazakov I applied the pkg change you requested. Please let me know if you have further concerns.

File: token/jwt/fetch_keys.go [hunk 2]
```diff
@@ -0,0 +1,98 @@
+package jwt
```
Comment (alexeykazakov) [inline token/jwt/fetch_keys.go]: Sorry @xcoulon but I said jwk. Not jwt.
JSON Web Key. Not JSON Web Token ;)

**Comment by alexeykazakov** [comment]
@xcoulon hm.. I can't push to your branch so I replaces your PR with this one - https://github.com/fabric8-services/fabric8-auth/pull/435
It's the same PR but with jwk package instead of jwt.

**Comment by xcoulon** [comment]
ah sorry for the misunderstanding, @alexeykazakov ! and thanks taking care of the change!

**Pull request closed without merging**